test/apiv2/01-basic.at

```diff
@@ -68,4 +68,8 @@ else
     _show_ok 0 "Time for ten /info requests" "<= 5 seconds" "$delta_t seconds"
 fi
 
+# Simple events test (see #7078)
+t GET "events?stream=false"  200
+t GET "libpod/events?stream=false"  200
+
```

Comment:
Friendly reminder to check actual results, not just status code. From a quick look at the returned JSON, might I suggest:
```bash
t GET "events?stream=false"        200 .Type=system .scope=local .time~[0-9]\\+
t GET "libpod/events?stream=false" 200 .Type=system .scope=local .time~[0-9]\\+
```

Reply:
Thanks for checking! It doesn't return any data here though. I really just want to exercise this specific code which we can't with bindings as the "filters" param is set unconditionally afaics.

Reply:
PS last week I looked into adding tests for `/events` but gave up because of the `follow` nature - I didn't want tests to hang. If you have hints for how to write non-hanging tests (looks like `?stream=false` is the magic token I was missing) I would like to look into that again. Thank you.

Reply:
Yes, `?stream=false` would be a good candidate for the tests here :+1:

Reply:
>It doesn't return any data here 

Weird! As in: when you run the test I suggested, it fails?? That alarms me a little bit: I tried my tests root and rootless.

Reply:
It now succeeds without `?filters=...`. It failed before this change with a regression I introduced. The other tests always set "filters", so here I just wanted to make sure that the code path is executed.